test/integration_tests/test_reuse.py

```diff
@@ -109,6 +119,10 @@ def test_reuse_does_not_work_when_expression_in_where_clause(self):
         select_query = """
             SELECT id FROM DETRAC
             WHERE ArrayCount(YoloV5(data).labels, 'car') > 3 AND id < 5;"""
+
```

Comment:
This should work now, right?

Reply:
Similarly, look at the query on line 112. That should also work now. Please enable it also. Thanks!